### DISCUSSION TITLE
오늘 참여가 어려울 것 같습니다.

### DISCUSSION THREAD

**Discussion by fkdl0048** [ThinkAboutSoftware/AcademicConference#271] [General]
생각보다 모임이 길어져서.. 오늘은 참여가 힘들 것 같습니다..

오늘 테스트코드 내용이라 이야기를 들어보고 싶었는데 너무 아쉽네요..ㅠ

**Comment by wisdom08** [comment]
그래서 못 오셨군요 아쉽네요 ㅜㅠ 😢 정안님 다음 모임 때 봬요!!
